apps/condo/domains/contact/hooks/useTableColumns.tsx

```diff
@@ -0,0 +1,120 @@
+import { Input, Space } from 'antd'
+import { Button } from '@condo/domains/common/components/Button'
+import { FilterValue } from 'antd/es/table/interface'
+import get from 'lodash/get'
+import { useIntl } from '@core/next/intl'
+import React, { useMemo } from 'react'
+import { FilterFilled } from '@ant-design/icons'
+import { colors } from '@condo/domains/common/constants/style'
+import { createSorterMap, IFilters } from '../utils/helpers'
+
+const getFilterIcon = filtered => <FilterFilled style={{ color: filtered ? colors.sberPrimary[5] : undefined }} />
+
+interface IFilterContainerProps {
+    clearFilters: () => void
+    showClearButton?: boolean
+}
+
+const FilterContainer: React.FC<IFilterContainerProps> = (props) => {
+    const intl = useIntl()
+    const ResetLabel = intl.formatMessage({ id: 'filters.Reset' })
+
+    return (
+        <div style={{ padding: 16 }}>
```

Comment:
I think its better to use emotion's css property when dealing with native browser elements. Inserting styles is more expensive during render time then inserting css class

We can rewrite this code like so:

```
import { css } from 'emotion'

const FilterContainerDiv = css`padding: 16`
```

Reply:
done 👍